File: Code Forces/734A-Anton-and-Danik.py
n=int(input())
s=(input())
ant=0
dan=0
for x in range (0,n) :
    if (s[x]=='A'):
        ant+=1
    elif (s[x]=='D'):
        dan+=1
if(ant>dan):
    print("Anton")
elif(ant<dan) :
    print("Danik")
elif(ant==dan) :
    print("Friendship")

File: Code Forces/734A-Anton-and-Danik.cpp
#include <iostream>

using namespace std;

int main()
{
    int n; //number of games played
    string g; //first letter of the winner of every game
    int a=0 ,d=0;
    cin>>n>>g;
    for (int i =0 ;i<n;i++)
    {
       if (g[i]=='A') a++;
       else if (g[i]=='D') d++;
    }
    if (a==d)cout<<"Friendship";
    else if (a>d)cout<<"Anton";
    else cout<<"Danik";
        return 0;
}